components/viz/service/main/viz_compositor_thread_runner_impl.cc

```diff
@@ -61,7 +61,7 @@ std::unique_ptr<VizCompositorThreadType> CreateAndStartCompositorThread() {
   if (!thread)
     thread = std::make_unique<base::Thread>(kThreadName);
 
-#if BUILDFLAG(IS_FUCHSIA)
+#if BUILDFLAG(IS_FUCHSIA) || BUILDFLAG(IS_STARBOARD)
```

Comment:
It looks to me like we should be using message_pump_type_for_viz_compositor above instead